### DISCUSSION TITLE
Always show buffers and tabs

### DISCUSSION THREAD

**Discussion by OBLIXi-ON** [NvChad/NvChad#2910] [Q&A] [answered]
Is there any way to always show buffer name and tab count even if there's just a single buffer and tab open?

**Comment by siduck** [comment, marked as answer]
set this to false in your chadrc https://github.com/NvChad/ui/blob/v2.5/lua/nvconfig.lua#L34

**Comment by danamkaplan** [comment]
For those referencing this in the future: @siduck referenced line 35 but that file has changed and 35 points to the wrong variable. What you want is `lazyload = false`, which currently is https://github.com/NvChad/ui/blob/v2.5/lua/nvconfig.lua#L34

TLDR; in your personal `.config/nvim/lua/chadrc.lua` - add:

```
M.ui = {
  tabufline = {
    lazyload = false,
  }
}
```

**Reply by OBLIXi-ON** [reply]
Thanks!
